### ISSUE TITLE
Hairpin with system breaks having problems

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#1905]
Hairpins that start on one system and end on another are having rendering problems. 

Here is an example where the crescendo is on a single system:

<img width="893" alt="Screen Shot 2020-12-23 at 3 02 01 PM" src="https://user-images.githubusercontent.com/3487289/103043122-a3930480-4530-11eb-8be6-58f58f84a27c.png">

<details>
<summary>
Click to see MEI data
</summary>

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt />
  </fileDesc>
  <encodingDesc>
   <appInfo>
    <application isodate="2020-12-23T15:01:02" version="3.1.0-dev-1629129-dirty">
     <name>Verovio</name>
     <p>Transcoded from Humdrum</p>
    </application>
   </appInfo>
  </encodingDesc>
  <workList>
   <work>
    <title />
   </work>
  </workList>
 </meiHead>
 <music>
  <body>
   <mdiv xml:id="mdiv-0000001475415278">
    <score xml:id="score-0000000010917172">
     <scoreDef xml:id="scoredef-0000000178133581" midi.bpm="400">
      <staffGrp xml:id="staffgrp-0000001085351900">
       <staffDef xml:id="staffdef-0000002016810264" n="1" lines="5">
        <clef xml:id="clef-0000001317074471" shape="G" line="2" />
        <meterSig xml:id="metersig-L2F1" count="4" unit="4" />
       </staffDef>
      </staffGrp>
     </scoreDef>
     <section xml:id="section-L1F1">
      <measure xml:id="measure-L1">
       <staff xml:id="staff-0000000484281517" n="1">
        <layer xml:id="layer-L1F1N1" n="1">
         <note xml:id="note-L4F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L5">
       <staff xml:id="staff-L5F1N1" n="1">
        <layer xml:id="layer-L5F1N1" n="1">
         <note xml:id="note-L6F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L7">
       <staff xml:id="staff-L7F1N1" n="1">
        <layer xml:id="layer-L7F1N1" n="1">
         <note xml:id="note-L8F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L9">
       <staff xml:id="staff-L9F1N1" n="1">
        <layer xml:id="layer-L9F1N1" n="1">
         <note xml:id="note-L10F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
       <dynam xml:id="dynam-L10F2" staff="1" tstamp="1.000000" vgrp="100">f</dynam>
       <hairpin xml:id="hairpin-L10F2" type="endbar03" staff="1" tstamp="1.000000" tstamp2="9m+1" form="cres" vgrp="100" />
      </measure>
      <measure xml:id="measure-L11">
       <staff xml:id="staff-L11F1N1" n="1">
        <layer xml:id="layer-L11F1N1" n="1">
         <note xml:id="note-L12F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L13">
       <staff xml:id="staff-L13F1N1" n="1">
        <layer xml:id="layer-L13F1N1" n="1">
         <note xml:id="note-L14F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L15">
       <staff xml:id="staff-L15F1N1" n="1">
        <layer xml:id="layer-L15F1N1" n="1">
         <note xml:id="note-L16F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L17">
       <staff xml:id="staff-L17F1N1" n="1">
        <layer xml:id="layer-L17F1N1" n="1">
         <note xml:id="note-L18F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L19">
       <staff xml:id="staff-L19F1N1" n="1">
        <layer xml:id="layer-L19F1N1" n="1">
         <note xml:id="note-L20F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L21">
       <staff xml:id="staff-L21F1N1" n="1">
        <layer xml:id="layer-L21F1N1" n="1">
         <note xml:id="note-L22F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L23">
       <staff xml:id="staff-L23F1N1" n="1">
        <layer xml:id="layer-L23F1N1" n="1">
         <note xml:id="note-L24F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L25">
       <staff xml:id="staff-L25F1N1" n="1">
        <layer xml:id="layer-L25F1N1" n="1">
         <note xml:id="note-L26F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L27">
       <staff xml:id="staff-L27F1N1" n="1">
        <layer xml:id="layer-L27F1N1" n="1">
         <note xml:id="note-L28F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L29">
       <staff xml:id="staff-L29F1N1" n="1">
        <layer xml:id="layer-L29F1N1" n="1">
         <note xml:id="note-L30F1" dur="1" oct="4" pname="g" accid.ges="n" />
        </layer>
       </staff>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

</details>

However, when there is a system break splitting the hairpin into multiple sections, the is a random behavior (probably related to an uninitialized variable):

<img width="499" alt="Screen Shot 2020-12-23 at 3 01 21 PM" src="https://user-images.githubusercontent.com/3487289/103043200-e8b73680-4530-11eb-9e30-aa7075f62aed.png">
<img width="628" alt="Screen Shot 2020-12-23 at 3 01 35 PM" src="https://user-images.githubusercontent.com/3487289/103043202-e9e86380-4530-11eb-927b-0becd8d55a96.png">
<img width="705" alt="Screen Shot 2020-12-23 at 3 01 48 PM" src="https://user-images.githubusercontent.com/3487289/103043203-eb199080-4530-11eb-8380-9c1e74bc662c.png">

This has been happening for around two weeks in the `develop` branch, I think.

**Comment by rettinghaus** [comment]
Was caused by #1876

**Comment by rettinghaus** [comment]
Should be fixed with #1906

**Comment by craigsapp** [comment]
Looks great!

<img width="814" alt="Screen Shot 2020-12-24 at 1 55 10 AM" src="https://user-images.githubusercontent.com/3487289/103080641-17fa9180-458b-11eb-988d-9b81a6213c5a.png">
